modules/geotiff/src/lib/load-geotiff.ts

```diff
@@ -8,29 +8,29 @@ import {
 // import Pool from './lib/Pool';
 
 import { loadOmeTiff, isOmeTiff } from './ome/load-ome-tiff';
+import type TiffPixelSource from './tiff-pixel-source';
 
-interface TiffOptions {
+/** Options for initializing a tiff pixel source. */
+interface GeoTIFFOptions {
+  /** Headers passed to each underlying request. */
   headers?: Record<string, unknown>;
+  /** Performance enhancment to index the remote tiff source using pre-computed byte-offsets. Generated via https://github.com/ilan-gold/generate-tiff-offsets */
   offsets?: number[];
+  /** Indicates whether a multi-threaded pool of image decoders should be used to decode tiles. */
   pool?: boolean;
 }
 
-/**
- * Opens an OME-TIFF via URL and returns data source and associated metadata for first image.
- *
- * @param {(string | Blob | GeoTIFF)} source url or File object.
- * @param {{ headers: (undefined | Headers), offsets: (undefined | number[]), pool: (undefined | boolean ) }} opts
- * Options for initializing a tiff pixel source. Headers are passed to each underlying fetch request. Offests are
- // * a performance enhancment to index the remote tiff source using pre-computed byte-offsets. Pool indicates whether a
- * multi-threaded pool of image decoders should be used to decode tiles (default = true).
- * @return {Promise<{ data: TiffPixelSource[], metadata: ImageMeta }>} data source and associated OME-Zarr metadata.
- */
```

Comment:
I think we still want the JSDoc comment itself, and the param description like `source url or File object` is still helpful. I think we just don't need to include the _types_ in both places.

Reply:
aka

```js
/**
 * Opens an OME-TIFF via URL and returns data source and associated metadata for first image.
 *
 * @param source url or File object.
 * @param opts
 * Options for initializing a tiff pixel source. Headers are passed to each underlying fetch request. Offests are
 // * a performance enhancment to index the remote tiff source using pre-computed byte-offsets. Pool indicates whether a
 * multi-threaded pool of image decoders should be used to decode tiles (default = true).
 * @return data source and associated OME-Zarr metadata.
 */
```

Reply:
Ah, ok. I split the comment to annotate the `GeoTIFFOptions` interface (above). I figured the source was self-explanatory (since the comment just reflects the type annotation).

Reply:
Either way, just wanted to make sure my initial suggestion wasn't interpreted as "remove all JSDoc comments".

Reply:
Yes, just remove the JSDoc types, not the comment. vscode shows JSDoc comments when hovering, which is really nice.